templates/docs/patterns/hero/index.md

```diff
@@ -17,12 +17,105 @@ The hero pattern is composed of the following elements:
 
 | Element              | Description                                                              |
 | -------------------- | ------------------------------------------------------------------------ |
-| Title (**required**) | <code>h1</code> title text                                               |
-| Subtitle             | <code>h2</code> subtitle text                                            |
-| Description          | <code>p</code> description text                                          |
+| Title (**required**) | `h1` title text                                                          |
+| Subtitle             | `h2` subtitle text                                                       |
+| Description          | `p` description text                                                     |
 | Call to action block | [Call to action block](/docs/patterns/cta-block) beneath the description |
 | Image                | Main hero visual                                                         |
 
+## Jinja Macro
+
+### Parameters
+
+<div style="overflow: auto;">
+  <table>
+    <thead>
+      <tr>
+        <th style="width: 190px;">Name</th>
+        <th style="width: 100px;">Is Required</th>
+        <th style="width: 200px;">Type</th>
+        <th style="width: 150px;">Default</th>
+        <th style="width: 250px;">Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><code>layout</code></td>
+        <td>Yes</td>
+        <td>
+          One of:
+          <ul class="p-list">
```

Comment:
Nesting bullet list inside table feels a bit off. Would a simple comma separated inline text work?

Reply:
It was originally a comma-separated list - @pastelcyborg [suggested](https://github.com/canonical/vanilla-framework/pull/5325#discussion_r1735321972) that each option be on its own line

Reply:
Changed to a comma-and-`<br>` separated field